web/themes/custom/vote_gov/templates/block/block-content--email-signup--full.html.twig

```diff
@@ -7,7 +7,7 @@
       {# what goes in action #}
         <form class="form-signup" action="{{ content.field_link | field_value }}" method="get">
           <h2 class="text-color-blue font-size-medium">
-              {{ content.field_heading | field_value | render | replace({'%USAGov%': 'USAGov'}) }}
+              {{ content.field_heading | field_value | render | replace({'%USAGov%': 'USAGov'}) | raw}}
```

Comment:
updated the text in the dev but also saw that it was giving the same issue as the nvrf and added this and its working on my local as well